### PR TITLE
Removing exception specifications

### PR DESCRIPTION
**Pull request by RedTaylor**
Removed all instances of exceptions from test files in csharp.

### CODE REVIEW

File: csharp/test/Ice/impl/Test.ice [hunk 1]
```diff
@@ -94,129 +94,89 @@ interface MyClass
 
     void opVoid();
 
-    byte opByte(byte p1, byte p2,
-                out byte p3);
+    (byte r1, byte r2) opByte(byte p1, byte p2);
```
Comment (pepone) [inline csharp/test/Ice/impl/Test.ice:97]: Changes in this file seems unrelated to the PR

File: csharp/test/Ice/optional/Test.ice [hunk 2]
```diff
@@ -56,15 +56,14 @@ module ZeroC::Ice::Test::Optional
     {
         void shutdown();
 
-        void opSingleInInt(int? i1);
-        void opSingleInString(string? i1);
-        int? opSingleOutInt();
-        string? opSingleOutString();
-        int? opSingleReturnInt();
-        string? opSingleReturnString();
+        void opInt(int? i1);
```
Comment (pepone) [inline csharp/test/Ice/optional/Test.ice:59]: Changes here and in other files of Ice/optional test seems unrelated to the PR

File: csharp/test/Slice/escape/Clash.ice [hunk 3]
```diff
@@ -24,9 +24,10 @@ interface Intf
 
     void op(string context, string current, string response, string ex, string sent, string cookie,
             string sync, string result, string istr, string ostr, tag(1) string? proxy, tag(2) int? obj);
-    void opOut(out string context, out string current, out string response, out string ex,
-               out string sent, out string cookie, out string sync, out string result, out string istr,
-               out string ostr, out tag(1) string? proxy, out tag(2) int? obj);
+    (string context, string current, string response,
```
Comment (pepone) [inline csharp/test/Slice/escape/Clash.ice:27]: Indentation here looks bogus and the changes is unrelated to removing exceptions specifications

File: csharp/test/Slice/escape/Key.ice [hunk 4]
```diff
@@ -64,24 +64,22 @@ interface taggedParams
...
-                     out tag(5) while? internal,
-                     out tag(7) string? namespace,
-                     out tag(8) explicit* null);
+    tag(1) break? catch(tag(2) as? goto,
+                        tag(3) explicit* if,
+                        tag(5) while? internal,
+                        tag(7) string? namespace,
+                        tag(8) explicit* null);
+
+    (tag(1) break? r1, tag(2) as? goto, tag(3) explicit* if,
```
Comment (pepone) [inline csharp/test/Slice/escape/Key.ice:73]: Indentation looks wrong here, and change is unrelated to this PR

**Review by pepone — CHANGES_REQUESTED** [review]
There are a few changes that are unrelated to the PR, is better if we just remove exception specifications here, and make the other changes separately

**Comment by RedTaylor** [comment]
Closing to fix branch

**Pull request closed without merging**